collections/St_Johns_College/SW_V1_St_Johns_College_MS_062.xml

```diff
@@ -0,0 +1,213 @@
+            <respStmt xml:id="RM">
+               <resp when="2002">Cataloguer</resp>
+               <persName>Ralph Hanna</persName>
+            </respStmt>
+            <respStmt xml:id="SJC">
+               <resp when="2021">Encoding</resp>
+               <persName>Sian Witherden</persName>
+            </respStmt>
+         </titleStmt>
+         <editionStmt>
+            <edition>TEI P5</edition>
+            <funder>Conversion of the printed catalogue to TEI funded by <orgName>the Thompson
+                  Family Charitable Trust</orgName></funder>
+         </editionStmt>
+         <publicationStmt>
+            <publisher>Special Collections, Bodleian Libraries</publisher>
+            <address>
+               <orgName type="department">Special Collections</orgName>
+               <orgName type="unit">Bodleian Libraries</orgName>
+               <orgName type="institution">University of Oxford</orgName>
+               <street>Weston Library, Broad Street</street>
+               <settlement key="place_7011931">Oxford</settlement>
+               <postCode>OX1 3BG</postCode>
+               <country>United Kingdom</country>
+            </address>
+            <distributor>
+               <email>specialcollections.enquiries@bodleian.ox.ac.uk</email>
+            </distributor>
+            <idno type="msID">St_Johns_College_MS_62</idno>
+            <idno type="collection">St_Johns_College</idno>
+            <idno type="catalogue">Western</idno>
+         </publicationStmt>
+         <sourceDesc>
+            <msDesc xml:lang="en" xml:id="St_Johns_College_MS_62">
+               <msIdentifier>
+                  <country>United Kingdom</country>
+                  <region type="county">Oxfordshire</region>
+                  <settlement key="place_7011931">Oxford</settlement>
+                  <institution>St John's College</institution>
+                  <repository>St John's College</repository>
+                  <idno type="shelfmark">St John's College MS. 62</idno>
+               </msIdentifier>
+               <msContents>
+                  <textLang mainLang="la">Latin</textLang>
+                  <msItem n="1" xml:id="St_Johns_College_MS_62-item1">
+                     <locus from="1ra" to="2vb">Fols. 1<hi rend="superscript">ra</hi>–2<hi
+                           rend="superscript">vb</hi></locus>
+                     <rubric>Incipiunt Cantica Canticorum que hebraice dicuntur Sy<gap
+                           unit="character" quantity="1"/>aserim Vox Synagoge cristi desiderantis
+                        aduentum</rubric>
+                     <incipit>Osculetur me osculo oris sui quia meliora sunt ubera</incipit>
+                     <explicit>ceruorum super montes aromatum</explicit>
+                     <finalRubric>Explicit cantica canticorum habent uersus <gap unit="character"
+                           quantity="5"/> ducentos Octogin</finalRubric>
```

Comment:
is Octogin correct? is something missing at the end?

Reply:
This is the spelling Hanna has in the printed catalogue. The only thing that follows in the catalogue is a close quotation mark and a full stop (both of which I omitted in line with our previous discussions). If you like, I can check his transcription against the manuscript the next time I am in the library?

Reply:
probably not worth the time!